serverless/serverless-architecture.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+
+// * serverless-overview.adoc
+
+[id="serverless-architecture_{context}"]
```

Comment:
@abrennan89 assemblies don't have context as an attribute to their anchor ids. See the assembly file structure: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata

Reply:
Neither of these should be assemblies now, think they're just in the wrong place. I moved them into modules.

Reply:
Sorry, the overview one is I guess, it includes the TP module.